src/components/ImgSlider/ImgSlider.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import ArrowSvg from '../../helper-components/ArrowSvg';
+import styles from './ImgSlider.module.css';
+
+const ImgSlider = ({ game }) => {
+  const white = '#FFFFFF';
+  const dirtyWhite = '#d1d1d1';
+  const [leftArrowNavFill, setLeftArrowNavFill] = React.useState('#d1d1d1');
+  const [rightArrowNavFill, setRightArrowNavFill] = React.useState('#d1d1d1');
+  const [activeImgIdx, setActiveImgIdx] = React.useState(0);
+
+  const displayNextImg = () => {
+    setActiveImgIdx((prevIdx) => {
+      if (prevIdx === 3) {
```

Comment:
What happens if one the games has more or less images? This slider should work with any number of images.